package.json

```diff
@@ -3,5 +3,9 @@
   "private": true,
   "scripts": {
     "publish-all": "node ./.scripts/publish-all"
+  },
+  "dependencies": {
```

Comment:
Do not make any changes to the root `package.json` file or the `package-lock.json` file. Create a new one inside `packages/clipboard`.